Formula/syslog-ng.rb

```diff
@@ -0,0 +1,51 @@
+class SyslogNg < Formula
+  desc "Log daemon with advanced processing pipeline and a wide range of I/O methods"
+  homepage "https://www.syslog-ng.com"
+
+  url "https://github.com/syslog-ng/syslog-ng/releases/download/syslog-ng-3.38.1/syslog-ng-3.38.1.tar.gz"
+  sha256 "5491f686d0b829b69b2e0fc0d66a62f51991aafaee005475bfa38fab399441f7"
+  license all_of: ["LGPL-2.1-or-later", "GPL-2.0-or-later"]
+
+  depends_on "autoconf" => :build
+  depends_on "autoconf-archive" => :build
+  depends_on "automake" => :build
+  depends_on "bison" => :build
+  depends_on "flex" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+
+  depends_on "curl"
+  depends_on "glib"
+  depends_on "hiredis"
+  depends_on "ivykis"
+  depends_on "json-c"
+  depends_on "libdbi"
+  depends_on "libmaxminddb"
+  depends_on "libnet"
+  depends_on "librdkafka"
+  depends_on "mongo-c-driver"
+  depends_on "net-snmp"
+  depends_on "openssl@1.1"
```

Comment:
```suggestion
  depends_on "openssl@3"
```

Reply:
OpenSSL v3 can not be used due to the following dependencies (they depend on 1.1):
- curl
- mongo-c-driver
- librdkafka
- net-snmp
- python@3.11